Entities/MOSRotating.cpp

```diff
@@ -738,6 +759,10 @@ void MOSRotating::AddRecoil()
 
 bool MOSRotating::CollideAtPoint(HitData &hd)
 {
+	if (m_ToDelete) {
+		return false;	// TODO: Add a settings flag to enable old school particle sponges!
```

Comment:
I forgot to add a comment for it, but there were also some TODOs in Arm (I think) that are in a similar situation. I'm cool with not dealing with them for now

Reply:
> I'm fine with this unaddressed, but I'm commenting as a reminder in case you wanna deal with this TODO.

Opened #492. Feel free to fix it if I've got anything wrong.